[CMSP-12] Multisite search-replace EA (#8303)
pantheon-systems/documentation: source/content/guides/multisite/05-workflows.md

To better understand what's going on, let's dive into `wp search-replace` with greater detail.

## Deploy Across Environments

<Alert title="Note" type="info">

> Multisite search and replace can now be handled automatically platform search & replace, currently in Early Access. For more details, see [Multisite Search Replace](/guides/multisite/search-replace).

Reviewer: ```suggestion
Multisite search and replace can now be handled automatically by platform search & replace, currently in Early Access. For more details, see [Multisite Search Replace](/guides/multisite/search-replace).
```

> Search Replace can now be handled automatically for Multisites with platform search-replace, currently in Early Access. For more details, see [Multisite Search Replace](/guides/multisite/search-replace).

Reviewer 2: Search & Replace
Multisite Search & Replace

Author: ```suggestion
Multisite search and replace can now be handled automatically platform search & replace, currently in Early Access. For more details, see [Multisite Search Replace](/guides/multisite/search-replace).
```
